lagunita/lms/templates/emails/activation_email.txt

```diff
@@ -0,0 +1,22 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
It's minor, but to keep these two files as in-sync as possible,
let's also include the mako comment;
it makes `diff`ing more in-line.

```
## mako
```